week-1-ts-ecosystem/w01e02-enter-the-world-of-typescript/src/interfaces/price.interface.ts

```diff
@@ -0,0 +1,5 @@
+export interface price {
+    value: number;
+    currency: 'PLN' | 'USD' | 'EUR' | 'GBP' ,
+  }
+  
```

Comment:
Widzę pomysł na cenę - i rozwój :)
Spoko. fajnie że dajesz @kubaGalinski interface'y osobno - widać co robią i jaki jest Twój model danych

Możesz jeszcze dać to jako `model` zamiast `interfaces` - jeśli chodzi o katalog.